cypress/integration/previewApi.spec.js

```diff
@@ -41,12 +47,4 @@ describe("Preview an API", () => {
                 }
             });
     });
-
-    it("When the user clicks on an API name they are directed to the SwaggerHub page", function() {
```

Comment:
I think we should keep this test

Reply:
I'll replace it with one to redirect to the API info page :)